samples/Build2019Demo.Unity/Assets/Demo/CustomProfiles/MixedRealityInputSystemProfile.asset

```diff
@@ -0,0 +1,88 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
```

Comment:
nit: It may be worth having this file have a build prefix to avoid confusion with other profiles